src/utils/cmp.js

```diff
@@ -39,15 +55,19 @@ const interactWithCMPCcpa = async (page) => {
 
 	await page.waitForNavigation({ waitUntil: 'domcontentloaded' });
 	// We see some run failures if we do not include a wait time after a page load
-	await page.waitForTimeout(TWO_SECONDS);
+	await new Promise((r) => setTimeout(r, TWO_SECONDS));
 };
 
 const interactWithCMPAus = async (page) => {
 	// When AWS Synthetics use a more up-to-date version of Puppeteer, we can make use of waitForFrame()
```

Comment:
Can we do what this comment says now? :p

Reply:
Yeah good shout - I did try this and couldn't get it work as reliably as what we already have. There is now a Playwright runtime for canaries though so that might be an option going forward 👀